_src/_inc/_macro/_anagrafica.form.php

```diff
@@ -65,15 +65,16 @@
 	    'SELECT id, html AS __label__ FROM tipologie_telefoni_view'
 	);
 
-/*
+
     // tendina tipologie indirizzi
 	$ct['etc']['select']['tipologie_indirizzi'] = mysqlCachedIndexedQuery(
 	    $cf['cache']['index'],
 	    $cf['memcache']['connection'],
 	    $cf['mysql']['connection'],
 	    'SELECT id, __label__ FROM tipologie_indirizzi_view'
 	);
-
```

Comment:
@silvia-df sulle tipologie di indirizzi sbaglio o avevamo un ragionamento in sospeso? non mi ricordo più, forse sulle icone?

Reply:
Avevo segnato di utilizzare la tendina delle tipologie, sulle icone avevamo avuto qualche perplessità perché diventa difficile poi distinguere, ad esempio, residenza da domicilio, ...

Reply:
ok valutiamo, per ora faccio il merge così